app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java

```diff
@@ -146,6 +147,10 @@ public void onCreate(Bundle savedInstanceState) {
         } else {
             loginCredentials.setVisibility(View.GONE);
         }
+
+
+
+
```

Comment:
Please remove unnecessary edits and spaces

Reply:
Thanks dear